### DISCUSSION TITLE
Unable to install zero-ui properly

### DISCUSSION THREAD

**Discussion by brokenby2703** [dec0dOS/zero-ui#117] [Q&A]
Hi everyone,
I need to deploy ZEROTIER on my own server.
I'm quite newbie on docker/docker-compose but I was able to setup my raspberry working.
For docker I did follow the original guide here: https://docs.docker.com/engine/install/debian/
For docker compose this one: https://docs.docker.com/compose/install/linux/
Both docker & docker-compose run perfectly.

I did follow exactly step by step your guide here: 'https://github.com/dec0dOS/zero-ui'

But when I try to run 'docker-compose up -d --no-build' , I can't connect to the webpage https://192.168.0.2:9993
There is no firewall installed. 

This is the output of the 'docker compose ps'

<img width="1244" alt="Screenshot 2022-10-16 at 13 41 48" src="https://user-images.githubusercontent.com/115927083/196033456-05d39f6c-ad14-4bd4-8577-705eadbacd46.png">

The 'docker-compose.yaml' is 100% the same from here : wget https://raw.githubusercontent.com/dec0dOS/zero-ui/main/docker-compose.yml

Anybody can help me please ?
Thanks

**Comment by russr999** [comment]
If you want to open zero-UI, I think you need to open **https://192.168.0.2/** or  **http://192.168.0.2** and caddy will proxy you to 192.168.0.2:4000
Also, on the docker-compose.yaml, did you enter your domain instead of _YOURDOMAIN.com_ ? Can everything inside/outside docker reach your domain name through DNs or setting in the hosts file?

**Reply by brokenby2703** [reply]
First of all, thank you so much for your fast reply.

I haven't changed the YOUR DOMAIN yet because actually I was thinking to access via IP only like http://70.0.*.*  which is a reachable server in VPS. So I will also disable the 

I currently testing the installation on my local (LAN) environment before deploy it in VPS server.

When I try Https://192.168.0.2 or http://192.168.0.2 I can't reach the login page. The wheel keep spinning but nothing happen.

**Reply by russr999** [reply]
caddy listens on ports 80 and 443. try just ignore caddy and goto zero-UI direct.  http://192.168.0.2:4000/app

**Comment by brokenby2703** [comment]
Sorry to bother you but I can't come out alive....

I tried to install in another 2 systems: RASPBERRY PI3 (Raspbian) and another SBC with Armbian OS

The same problem.
I tried any possible combination.... http/https to 80,443,4000 with or without /app... nothing happen all returned 'page not found'

I'm really willing to give ssh access to anyone who can troubleshoot why I mistake....

**Comment by brokenby2703** [comment]
Update!
Managed to install and make it running on a VPS (linode)
I created a domain and pointed to the IP of the VPS
It works !
I also changed the outside ports 443 and 80 to 60443 and 60080.

But when I do the same in my LAN.... it doesn't work.
I port forwarded the ports 60443 and 60080 to the Raspberry.
I updated the IP in the duckpins.org to my IP but again... not working.
